includes/Story_Post_Type.php

```diff
@@ -254,7 +258,7 @@ public static function filter_use_block_editor_for_post_type( $use_block_editor,
 	 *
 	 * @return void
 	 */
-	public static function admin_enqueue_scripts( $hook ) {
+	public function admin_enqueue_scripts( $hook ) {
```

Comment:
The `$settings = self::get_editor_settings();` call further down can be replaced with `$settings = self::get_editor_settings();`

Reply:
Fixed https://github.com/google/web-stories-wp/pull/2595/commits/88733befb0e22cb269bdf2cad75b7c249ce7aeb4